Modules/_testcapi/heaptype.c

```diff
@@ -365,6 +365,21 @@ create_type_from_repeated_slots(PyObject *self, PyObject *variant_obj)
 }
 
 
+
+static PyObject *
+make_immutable_type_with_base(PyObject *self, PyObject *base)
+{
+    assert(PyType_Check(base));
+    PyType_Spec ImmutableSubclass_spec = {
+        .name = "ImmutableSubclass",
+        .basicsize = ((PyTypeObject*)base)->tp_basicsize,
```

Comment:
You've probably already seen it, but this needs type cast to `int` to get rid of the compiler warning.